expression/integration_serial_test.go

```diff
@@ -2043,10 +2043,8 @@ func TestTimeBuiltin(t *testing.T) {
 	result = tk.MustQuery(`select quarter("2012-14-20"), quarter("aa"), quarter(null), quarter(11), quarter(12.99);`)
 	result.Check(testkit.Rows("<nil> <nil> <nil> <nil> <nil>"))
 	result = tk.MustQuery(`select quarter("0000-00-00"), quarter("0000-00-00 00:00:00");`)
-	result.Check(testkit.Rows("<nil> <nil>"))
-	tk.MustQuery("show warnings").Check(testutil.RowsWithSep("|",
-		"Warning|1292|Incorrect datetime value: '0000-00-00 00:00:00.000000'",
-		"Warning|1292|Incorrect datetime value: '0000-00-00 00:00:00.000000'"))
+	result.Check(testkit.Rows("0 0"))
```

Comment:
This test is running under allow zero date sql mode?

Reply:
Yes, see here https://github.com/pingcap/tidb/pull/31078/files#diff-3b513c8044c3b047af40bb879b7e650a636ee695d5740881f8d861c52fa96c60L1990